sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -199,7 +199,7 @@ statement
     | SHOW CREATE TABLE multipartIdentifier                            #showCreateTable
     | SHOW CURRENT NAMESPACE                                           #showCurrentNamespace
     | (DESC | DESCRIBE) FUNCTION EXTENDED? describeFuncName            #describeFunction
-    | (DESC | DESCRIBE) database EXTENDED? db=errorCapturingIdentifier #describeDatabase
+    | (DESC | DESCRIBE) database EXTENDED? db=multipartIdentifier      #describeDatabase
```

Comment:
let's be consistent with `CREATE (database | NAMESPACE)`, and also use `(database | NAMESPACE)` here.

Reply:
also remove `db=` as it's a namespace name now.

Reply:
also update the label `#describeDatabase` to `#describeNamespace`